README

```diff
@@ -87,12 +87,14 @@ An earlier version (PostgreSQL 9) might be:
     
 Then run:
 
+    ./autogen.sh
```

Comment:
This would mean to suggest users to recreate everything - this _really_ should only be done when checking out from version control, while "normal users" should be provided a distribution tarball where this isn't necessary.

Note: the distribution tarball can (and should) be made available in the CI runner as artifact, too.